invenio_cli/commands/services_health.py

```diff
@@ -49,7 +49,7 @@ def postgresql_healthcheck(cls, *args, **kwargs):
                 "exec",
                 "-T",
                 "db",
-                "bash",
+                "sh",
```

Comment:
The idea here being that one may have modified their docker-compose.yml file to use an alpine based image for their postgresql or redis service... With `sh` being on both images this works.

Reply:
yes. `bash` is to heavy for `alpine` ;)